Syntax/Just.sublime-syntax

```diff
@@ -507,8 +507,8 @@ contexts:
       scope: storage.modifier.definition.just
       push:
         - settings-invalid
-        - settings-boolean
         - settings-shell
+        - settings-boolean
         - settings-string
```

Comment:
I don't think multi-push is the appropriate solution here.

What happens?

1. All contexts are pushed onto stack at once.
2. It tries to find a `string` setting. 
   - If not found, popping and continue trying with bool. etc. That's what you probably expect to happen.
   - If string matches pushing to `assignment-operator`, popping back, consuming any number of quoted strings, popping, matching shell settings.

Here's a screenshot with a bool followed by a shell without `set` keyword needed. I don't think that's intented.

![grafik](https://user-images.githubusercontent.com/16542113/235988713-9f76675d-db9f-4846-a03b-fa18100e4d3e.png)